spec/acceptance/basic_candlepin_spec.rb

```diff
@@ -10,6 +10,15 @@
   end
 
   describe command('curl -k -s -o /dev/null -w \'%{http_code}\' https://localhost:8443/candlepin/status') do
-      its(:stdout) { should eq "200" }
+    its(:stdout) { should eq "200" }
+  end
+
+  describe command('nmap --script +ssl-enum-ciphers localhost -p 8443') do
+    its(:stdout) do
+      should_not match(/TLSv1\.3/)
+      should match(/TLSv1\.2/)
+      should_not match(/TLSv1\.1/)
+      should_not match(/TLSv1\.0/)
+    end
```

Comment:
I'd use multiple `its(:stdout)` blocks. That way, if one fails, you can still see the output of the other ones. Multiple assertions in a single block is an anti pattern. We use it in some places because compiling the full Puppet module can be expensive. Here we just look at a string so it's cheap.

Reply:
Thanks for this by the way. I saw this issue in the test output and meant to come back and figure out how to fix it. Your explanation makes perfect sense.